### PR TITLE
docs: improved recommended gitignore list

### PR DESCRIPTION
**Pull request by Null78**
files that are important to be ignored also in a nuxt project

### ❓ Type of change

- [x] 📖 Documentation (updates to the documentation, readme or JSdoc annotations)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [ ] ✨ New feature (a non-breaking change that adds functionality)
- [ ] 🧹 Chore (updates to the build process or auxiliary tools and libraries)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

added important files and folders to ignore in a nuxt project

### 📝 Checklist

- [ ] I have linked an issue or discussion.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: docs/2.guide/2.directory-structure/2.gitignore.md [hunk 1]
```diff
@@ -15,8 +15,20 @@ We recommend having a `.gitignore` file that has **at least** the following entr
...
 *.log
+
+# Misc
+.DS_Store
+
+# Local env files
+.env.local
+.env.development.local
+.env.test.local
+.env.production.local
```
Comment (danielroe) [inline docs/2.guide/2.directory-structure/2.gitignore.md:33]:
```suggestion
.env.*
!.env.example
```

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Would you be able to make a PR to https://github.com/nuxt/starter to align the recommendation and the `.gitignore` we ship?

**Comment by Null78** [comment]
ok i'll open a new PR to https://github.com/nuxt/starter ✅

**Comment by Null78** [comment]
i was thinking about adding IDE specific files like `.vs-code` and `.idea`
should i add it to both repos?

**Comment by danielroe** [comment]
But I think not .vscode as this is often intentionally committed

**Comment by Null78** [comment]
ok so no need to add it ✅

**Comment by Null78** [comment]
also here is the PR to the starters repo
https://github.com/nuxt/starter/pull/395

**Review by danielroe — APPROVED** [review]

**Comment by danielroe** [comment]
Thank you! ❤️

**Pull request merged**